README.md

```diff
@@ -3,8 +3,104 @@
 Keto is under development and should not be used or treated as working
 software. Everything will probably change.
 
-### Building
+## Pre-requisites
 
+### Install golang
+
+Keto is built against the latest golang which can be installed from https://golang.org/dl/.
```

Comment:
Worth mentioning what the current Golang version used is? (Am thinking just in case over time a new Golang version is no longer compatible.